src/core/controllers/map-controls.js

```diff
@@ -38,36 +34,4 @@ export default class MapControls extends ViewportControls {
   _onPan(event) {
     return this.isFunctionKeyPressed(event) ? this._onPanRotate(event) : this._onPanMove(event);
   }
-
```

Comment:
without default _onPanRotate(event) function, do we have one in the parent class?

Reply:
Yes